src/util/DateUtils.ts

```diff
@@ -176,8 +176,7 @@ export class DateUtils {
 
     static stringToSimpleJson(value: any) {
         try {
-            const simpleJSON = JSON.parse(value);
-            return (typeof simpleJSON === "object") ? simpleJSON : {};
+            return JSON.parse(value);
```

Comment:
https://github.com/typeorm/typeorm/pull/5125#issuecomment-572992340 proposes to return `{}` when `value` is empty. However, that means `0` `''` and `null` would never be a valid value, when those are indeed valid JSON.

Reply:
`value` should be the raw JSON string coming from the DB. This should mean that it is only falsey when the field is set to `NULL` or the empty string.

`0`, `""` and `null` should be strings of non-zero length, allowing them to be safely read and stored.